plugin/mocha.vim

```diff
@@ -63,7 +63,7 @@ function! s:GetNearestTest()
   let file = readfile(expand("%:p"))  "read current file
   let lineCount = 0                   "file line counter
   let lineDiff = 999                  "arbituary large number
-  let descPattern='\v\s*it\s*[(]?\s*([''"]{1})(.+)\1{1}'
+  let descPattern = '\v<(it|describe|context)\s*\(?\s*[''"](.*)[''"]\s*,\s*(function|-\>)'
```

Comment:
Not sure if you are familiar with ES6, but the last part of this pattern - `,\s*(function|-\>)` - wouldn't handle [ES6](http://babeljs.io/docs/learn-es2015/#arrows) or [async tests](http://mochajs.org/#asynchronous-code). No need to worry around the ES6 portion, but the reason this part was absent in the old pattern was so it can handle async tests.